modules/claims_api/app/views/claims_api/_submission_grouped_table.html.erb

```diff
@@ -7,11 +7,9 @@
   </thead>
   <tbody>
     <% claims&.each_with_index do |grps, idx| %>
-      <% grps[1]&.each do |arr| %>
           <tr>
-            <td><%= grps[0] %></td><td><%= arr[:id] %></td>
+            <td><%= grps[0] %></td><td><%= grps[1].pluck(:id) %></td>
```

Comment:
Same as above - more descriptive variable names may help us with maintenance.

Reply:
Same as above

Reply:
How about `grps` → `transaction_group`? And could we remove `with_index` and `idx` since they are unused?